src/App.js

```diff
@@ -28,8 +28,13 @@ import { CLIENT_DEFAULTS } from './utils/CLIENT_DEFAULTS';
 // LOCALIZATION
 import { getTextForLanguage } from './utils/getTextForLanguage';
 
+/** App component; main top-level component of the app */
 // Change HashRouter tags below to Router tags to turn off hash routing; only used to be compatible with GitHub Pages
 class App extends Component {
+  /**
```

Comment:
This should probably be above the class declaration, like the other function descriptions. Unless someone knows otherwise.

Reply:
Ah, this is the description of the constructor, as your comments above explain. Nvm.